content/en/docs/use-cases/karmada.md

```diff
@@ -0,0 +1,193 @@
+
+```sh
+git clone https://github.com/karmada-io/karmada
+cd karmada
+hack/local-up-karmada.sh
+```
+
+After that, you will start a Kubernetes cluster by kind to run the Karmada control plane and create member clusters managed by Karmada.
+
+```sh
+kubectl get clusters --kubeconfig=/$HOME/.kube/karmada.config
+```
+
+you can use the command above to check registered clusters, and you will get similar output as follows:
+
+```
+NAME      VERSION   MODE   READY   AGE
+member1   v1.23.4   Push   True    7m38s
+member2   v1.23.4   Push   True    7m35s
+member3   v1.23.4   Pull   True    7m27s
+```
+
+### Flux Installation 
+
+In Karmada control plane, you only need to install Flux crds [here](https://github.com/fluxcd/flux2/blob/main/manifests/crds/kustomization.yaml) as objects delivered to member clusters.
+
+In member clusters, you can install Flux without storing its manifests in a Git repository in member clusters for testing purposes:
+
+```sh
+flux install
+```
+
+Please refer to the documentations [here](../installation.md) for more ways to set up Flux in details.
+
+Tips:
+
+1. The Flux toolkit components need to be installed in each cluster if you want to manage helm charts in each cluster.
+
+## Helm chart propagation
+
+If you want to propagate helm applications to member clusters, you can refer to the guide below.
+
+1. Define a HelmRepository manifest and a HelmRelease manifest in Karmada Control plane. They will serve as a resource template. 
+
+```yaml
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: HelmRepository
+metadata:
+  name: podinfo
+spec:
+  interval: 1m
+  url: https://stefanprodan.github.io/podinfo  
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: podinfo
+spec:
+  interval: 5m
+  chart:
+    spec:
+      chart: podinfo
+      version: 5.0.3
+      sourceRef:
+        kind: HelmRepository
+        name: podinfo
+```
+
+2. Define a PropagationPolicy that will propagate them to member clusters:
+
+```yaml
+apiVersion: policy.karmada.io/v1alpha1
+kind: PropagationPolicy
+metadata:
+  name: helm-repo
+spec:
+  resourceSelectors:
+    - apiVersion: source.toolkit.fluxcd.io/v1beta2
+      kind: HelmRepository
+      name: podinfo
+  placement:
+    clusterAffinity:
+      clusterNames:
+        - member1
+        - member2
+---
+apiVersion: policy.karmada.io/v1alpha1
+kind: PropagationPolicy
+metadata:
+  name: helm-release
+spec:
+  resourceSelectors:
+    - apiVersion: helm.toolkit.fluxcd.io/v2beta1
+      kind: HelmRelease
+      name: podinfo
+  placement:
+    clusterAffinity:
+      clusterNames:
+        - member1
+        - member2
+```
+
+It means that the HelmRepository and HelmRelease created in Step 1 will be propagated to member1 and member2.
+
+3. Apply those manifests to the Karmada-apiserver:
+
+```sh
+kubectl apply -f ../helm/ --kubeconfig=/$HOME/.kube/karmada.config
+```
+
+The output is similar to:
+
+```
+helmrelease.helm.toolkit.fluxcd.io/podinfo created
```

Comment:
How come these Flux custom resources work on this cluster? Isn't the controller plane cluster getting filled with Helm release which shouldn't be reconcile here?